lib/fund.js

```diff
@@ -37,161 +32,174 @@ fundCmd.completion = function (opts, cb) {
   }
 }
 
+const cmd = (args, cb) => fund(args).then(() => cb()).catch(cb)
+
 function printJSON (fundingInfo) {
   return JSON.stringify(fundingInfo, null, 2)
 }
 
-// the human-printable version does some special things that turned out to
-// be very verbose but hopefully not hard to follow: we stack up items
-// that have a shared url/type and make sure they're printed at the highest
-// level possible, in that process they also carry their dependencies along
-// with them, moving those up in the visual tree
-function printHuman (fundingInfo, opts) {
-  // mapping logic that keeps track of seen items in order to be able
-  // to push all other items from the same type/url in the same place
-  const seen = new Map()
-
-  function seenKey ({ type, url } = {}) {
-    return url ? String(type) + String(url) : null
-  }
-
-  function setStackedItem (funding, result) {
-    const key = seenKey(funding)
-    if (key && !seen.has(key)) seen.set(key, result)
-  }
-
-  function retrieveStackedItem (funding) {
-    const key = seenKey(funding)
-    if (key && seen.has(key)) return seen.get(key)
-  }
+const getPrintableName = ({ name, version }) => {
+  const printableVersion = version ? `@${version}` : ''
+  return `${name}${printableVersion}`
+}
 
-  // ---
-
-  const getFundingItems = (fundingItems) =>
-    Object.keys(fundingItems || {}).map((fundingItemName) => {
-      // first-level loop, prepare the pretty-printed formatted data
-      const fundingItem = fundingItems[fundingItemName]
-      const { version, funding } = fundingItem
-      const { type, url } = funding || {}
-
-      const printableVersion = version ? `@${version}` : ''
-      const printableType = type && { label: `type: ${funding.type}` }
-      const printableUrl = url && { label: `url: ${funding.url}` }
-      const result = {
-        fundingItem,
-        label: fundingItemName + printableVersion,
+// the human-printable version does some special things that turned out to be
+// verbose but hopefully not hard to follow: we stack up items that have
+// a shared url and make sure they're printed at the highest level possible
+function printHuman (fundingInfo, opts) {
+  const { name, version } = fundingInfo
+  const header = getPrintableName({ name, version })
+  const result = {}
+  const seenUrls = new Map()
+  let prev = fundingInfo
+  let parent = result
+
+  const tree = obj =>
+    archy(obj, '', { unicode: opts.unicode })
```

Comment:
This could be `archy(obj, '', {unicode})` if the function definition was `function printHuman (fundingInfo, {unicode})` :)